planemo/github_util.py

```diff
@@ -55,12 +56,13 @@ def clone_fork_branch(ctx, target, path, **kwds):
             pass
 
 
-def fork(ctx, path, **kwds):
+def fork(ctx, path, remote_name=DEFAULT_REMOTE_NAME, **kwds):
     """Fork the target repository using ``hub``."""
     gh_path = ensure_gh(ctx, **kwds)
     gh_env = get_gh_env(ctx, path, **kwds)
-    cmd = [gh_path, "repo", "fork"]
+    cmd = [gh_path, "repo", "fork", '--remote', '--remote-name', remote_name]
```

Comment:
```suggestion
    cmd = [gh_path, "repo", "fork", '--remote', 'true', '--remote-name', remote_name]
```

Reply:
The `remote` flag doesn't take a value in 1.5.